data/azure/instance_types.json

```diff
@@ -97,18 +120,126 @@
+  "Standard_E20d_v4": {"up": "Standard_E32d_v4", "down": "Standard_E16d_v4", "superseded": null},
+  "Standard_E32d_v4": {"up": "Standard_E48d_v4", "down": "Standard_E20d_v4", "superseded": null},
+  "Standard_E48d_v4": {"up": "Standard_E64d_v4", "down": "Standard_E32d_v4", "superseded": null},
+  "Standard_E64d_v4 ": {"up": null, "down": "Standard_E48d_v4", "superseded": null},
   "Standard_E2s_v3": {"up": "Standard_E4s_v3", "down": null, "superseded": null},
   "Standard_E4s_v3": {"up": "Standard_E8s_v3", "down": "Standard_E2s_v3", "superseded": null},
   "Standard_E8s_v3": {"up": "Standard_E16s_v3", "down": "Standard_E4s_v3", "superseded": null},
-  "Standard_E16s_v3": {"up": "Standard_E32s_v3", "down": "Standard_E8s_v3", "superseded": null},
-  "Standard_E32s_v3": {"up": "Standard_E64s_v3", "down": "Standard_E16s_v3", "superseded": null},
-  "Standard_E64s_v3": {"up": null, "down": "Standard_E32s_v3", "superseded": null},
+  "Standard_E16s_v3": {"up": "Standard_E20s_v3", "down": "Standard_E8s_v3", "superseded": null},
+  "Standard_E20s_v3": {"up": "Standard_E32s_v3", "down": "Standard_E16s_v3", "superseded": null},
+  "Standard_E32s_v3": {"up": "Standard_E48s_v3", "down": "Standard_E20s_v3", "superseded": null},
+  "Standard_E48s_v3": {"up": "Standard_E64s_v3", "down": "Standard_E32s_v3", "superseded": null},
+  "Standard_E64s_v3": {"up": null, "down": "Standard_E48s_v3", "superseded": null},
+  "Standard_E16-4s_v3": {"up": "Standard_E16-8s_v3", "down": null, "superseded": {"regular": "Standard_E16s_v3"}},
+  "Standard_E16-8s_v3": {"up": null, "down": "Standard_E16-4s_v3", "superseded": {"regular": "Standard_E16s_v3"}},
+  "Standard_G1": {"up": "Standard_G2", "down": null, "superseded": null},
+  "Standard_G2": {"up": "Standard_G3", "down": "Standard_G1", "superseded": null},
+  "Standard_G3": {"up": "Standard_G4", "down": "Standard_G2", "superseded": null},
+  "Standard_G4": {"up": "Standard_G5", "down": "Standard_G3", "superseded": null},
+  "Standard_G5": {"up": null, "down": "Standard_G4", "superseded": null},
+  "Standard_GS1": {"up": "Standard_GS2", "down": null, "superseded": null},
+  "Standard_GS2": {"up": "Standard_GS3", "down": "Standard_GS1", "superseded": null},
+  "Standard_GS3": {"up": "Standard_GS4", "down": "Standard_GS2", "superseded": null},
+  "Standard_GS4": {"up": "Standard_GS5", "down": "Standard_GS3", "superseded": null},
+  "Standard_GS5": {"up": null, "down": "Standard_GS4", "superseded": null},
+  "Standard_L4s": {"up": "Standard_L8s", "down": null, "superseded": null},
+  "Standard_L8s": {"up": "Standard_L16s", "down": "Standard_L4s", "superseded": null},
+  "Standard_L16s": {"up": "Standard_L32s", "down": "Standard_L8s", "superseded": null},
+  "Standard_L32s": {"up": null, "down": "Standard_L16s", "superseded": null},
+  "Standard_L8s_v2": {"up": "Standard_L16s_v2", "down": null, "superseded": null},
+  "Standard_L16s_v2": {"up": "Standard_L32s_v2", "down": "Standard_L4s", "superseded": null},
```

Comment:
The value for "down" should be "Standard_L8s_v2" instead of "Standard_L4s"

Reply:
Done